Makefile

```diff
@@ -1,15 +1,17 @@
 SPEC=PNA
+ROUGE_STYLE=github
 
-all: build/${SPEC}.pdf
+all: ${SPEC}.pdf
 
-build/${SPEC}.pdf: ${SPEC}.mdk
-	madoko --pdf -vv --png --odir=build $<
-
-build/${SPEC}.pdf: p4.json
-build/${SPEC}.pdf: pna.p4
+${SPEC}.pdf: ${SPEC}.adoc
```

Comment:
Please add the file pna.p4 as a dependency on this rule, and the rule for generating HTML output, since PNA.adoc has include directives that include portions of pna.p4.  If pna.p4 changes, it is likely that the PDF and/or HTML files need to be updated, too.

Reply:
Also, please add a Makefile rule to generate HTML output.  I don't see one here.  It would be helpful to have that, even if the original Makefile did not have such a rule to generate HTML from Madoko source.